pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -216,13 +216,13 @@ def on_train_epoch_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningMo
             # performing only one pass over the train data-loader to compute activation statistics
             # Therefore, we will virtually increase `num_training_batches` by 1 and skip backward.
             trainer.num_training_batches += 1
-            trainer.fit_loop._skip_backward = True
+            trainer.fit_loop.epoch_loop.batch_loop.optimizer_loop._skip_backward = True
```

Comment:
Why not use the property in `fit_loop` for it?

```suggestion
            trainer.fit_loop._skip_backward = True
```

Otherwise remove the property itself

Reply:
i was anticipating this comment.
on the one hand skip_backward has nothing to do with the fit loop
on the other hand we don't want to reach so deep into the loop stack, it  becomes fragile when opening up the loops for customization.

related: #9220